components/SwipeRow.js

```diff
@@ -134,24 +140,35 @@ class SwipeRow extends Component {
 		}
 	}
 
-	handlePanResponderEnd(e, gestureState) {
-		// re-enable scrolling on listView parent
+	ensureScrollEnabled() {
 		if (!this.parentScrollEnabled) {
 			this.parentScrollEnabled = true;
 			this.props.setScrollEnabled && this.props.setScrollEnabled(true);
 		}
+	}
+
+	handlePanResponderEnd(e, gestureState) {
+
+		// decide how much the velocity will affect the final position that the list item settles in.
+		const swipeToOpenVelocityContribution = this.props.swipeToOpenVelocityContribution;
+		const possibleExtraPixels = this.props.rightOpenValue * (swipeToOpenVelocityContribution);
+		const clampedVelocity = gestureState.vx > 5 ? 5 : gestureState.vx
+		const projectedExtraPixels = possibleExtraPixels * (clampedVelocity / 5);
```

Comment:
can we move the 5 to the top of the file as a `const`. i.e.
```
const MAX_VELOCITY_CONTRIBUTION = 5;
```

or something like that

Reply:
yes.